server/routes/v1/deployment.ts

```diff
@@ -111,3 +113,28 @@ export const postDeployment = [
     })
   },
 ]
+
+export const resetDeploymentApprovals = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    const user = req.user
+    const body = req.body as any
+    const deployment: Deployment = await DeploymentModel.findOne({ uuid: body.uuid })
```

Comment:
`{ uuid: body.uuid }`, mentioned above it makes no sense to send entire document.  Instead, take this from `req.params.uuid`.

Reply:
Resolved